### PR TITLE
feat(app): remove Change pipette ff

### PR DESCRIPTION
**Pull request by jerader**
closes RQA-261

# Overview

This removes the change pipette ff and deletes all the deprecated components. The `ChangePipette` parent component renders the OT-2 attach and detach flows.

# Changelog

- removes the `ChangePipette` FF and deletes all deprecated components
- fixes a small bug I noticed in the `ConfirmPipette` page where the header was saying the incorrect info

# Review requests

- Walk through the attach pipette flow for an OT-2 pipette, should work as expected including the buttons and early exit buttons and error handling
- walk through the detach pipette flow for an OT-2 pipette, should work as expected including the buttons and early exit buttons and error handling

# Risk assessment

low

### CODE REVIEW

**Comment by smb2268** [comment]
this diff 🤩
Screen Shot 2022-12-13 at 1 36 02 PM

File: app/src/organisms/ChangePipette/index.tsx [hunk 1]
```diff
@@ -338,17 +284,22 @@ export function ChangePipette(props: Props): JSX.Element | null {
...
-          })
-        : t('attach_name_pipette', {
-            pipette:
-              wrongWantedPipette != null
-                ? wrongWantedPipette.displayName
-                : wantedPipette?.displayName,
-          })
+    let wizardTitleConfirmPipette
+    if (wantedPipette == null && actualPipette == null) {
+      wizardTitleConfirmPipette = t('detatch_pipette_from_mount', {
```
Comment (koji) [inline app/src/organisms/ChangePipette/index.tsx]: probably
```suggestion
      wizardTitleConfirmPipette = t('detach_pipette_from_mount', {
```

File: app/src/assets/localization/en/change_pipette.json [hunk 2]
```diff
@@ -47,5 +47,7 @@
   "calibrate_pipette_offset": "Calibrate pipette offset",
   "try_again": "Try again",
   "detatch_try_again": "Detach and try again",
-  "detatch_pipette_from_mount": "Detach Pipette from {{mount}} Mount"
+  "detatch_pipette_from_mount": "Detach Pipette from {{mount}} Mount",
```
Comment (koji) [inline app/src/assets/localization/en/change_pipette.json]:
```suggestion
  "detach_pipette_from_mount": "Detach Pipette from {{mount}} Mount",
```

File: app/src/assets/localization/en/change_pipette.json [hunk 3]
```diff
@@ -47,5 +47,7 @@
   "calibrate_pipette_offset": "Calibrate pipette offset",
   "try_again": "Try again",
   "detatch_try_again": "Detach and try again",
```
Comment (koji) [inline app/src/assets/localization/en/change_pipette.json]:
```suggestion
  "detach_try_again": "Detach and try again",
```

File: app/src/organisms/ChangePipette/__tests__/CheckPipettesButton.test.tsx [hunk 4]
```diff
@@ -47,29 +42,19 @@ describe('CheckPipettesButton', () => {
...
 
-  it('renders the button without ff and clicking on it calls fetchPipettes', () => {
-    mockFeatureFlag.mockReturnValue(false)
-    const { getByRole } = render(props)
-    const btn = getByRole('button', { name: 'btn text' })
-    fireEvent.click(btn)
-    expect(dispatchApiRequest).toBeCalledWith(mockFetchPipettes('otie'))
-  })
-  it('renders the confirm attachment btn with ff and clicking on it calls fetchPipettes', () => {
+  it('renders the confirm attachment btn  and clicking on it calls fetchPipettes', () => {
```
Comment (koji) [inline app/src/organisms/ChangePipette/__tests__/CheckPipettesButton.test.tsx]: seems that after btn there is an extra space in the test description. 
```suggestion
  it('renders the confirm attachment btn and clicking on it calls fetchPipettes', () => {
```

**Review by koji — APPROVED** [review]
left a couple of comments but the changes look good to me!
 🧹 🧹 🧹

**Review by smb2268 — APPROVED** [review]
This looks good!! I tested it briefly on an OT-2 and it all worked as expected.

File: app/src/organisms/ChangePipette/index.tsx (hunk 1 above)
Comment (jerader) [reply app/src/organisms/ChangePipette/index.tsx]: thank you for catching this!

File: app/src/assets/localization/en/change_pipette.json (hunk 3 above)
Comment (jerader) [reply app/src/assets/localization/en/change_pipette.json]: thank you for catching this as well!!

**Pull request merged**